nekoyume/Assets/_Scripts/UI/Widget/Popup/ChatPopup.cs

```diff
@@ -18,7 +18,6 @@ public override void Initialize()
             {
                 Application.OpenURL(GameConfig.DiscordLink);
                 HelpTooltip.HelpMe(100012, true);
-                AudioController.PlayClick();
```

Comment:
Is this intended?

Reply:
I set this event in inspector. So removed several similar parts.